### PR TITLE
network_layer: remove last occurences of vtimer

### PR DESCRIPTION
**Pull request by cgundogan**
This PR removes further occurences of vtimer by replacing them with xtimer in `gnrc_icmpv6`, `gnrc_ipv6`, `gnrc_ndp` and `gnrc_sixlowpan`

### CODE REVIEW

**Comment by authmillenon** [comment]
Can you also replace the last occurrences of `timex_t` while you are at it?

File: sys/net/gnrc/network_layer/ipv6/nc/gnrc_ipv6_nc.c [hunk 1]
```diff
@@ -24,7 +24,6 @@
 #include "net/gnrc/sixlowpan/nd.h"
 #include "thread.h"
 #include "timex.h"
-#include "vtimer.h"
```
Comment (authmillenon) [inline sys/net/gnrc/network_layer/ipv6/nc/gnrc_ipv6_nc.c]:
```diff
+#include "xtimer.h"
```
?

(below too?)

Comment (cgundogan) [inline sys/net/gnrc/network_layer/ipv6/nc/gnrc_ipv6_nc.c]: xtimer.h gets included by other header files AFAICS and is not needed here

**Comment by cgundogan** [comment]
>Can you also replace the last occurrences of timex_t while you are at it?

sure, will do that in a follow-up PR

File: sys/net/gnrc/network_layer/ipv6/nc/gnrc_ipv6_nc.c (hunk 1 above)
Comment (authmillenon) [inline sys/net/gnrc/network_layer/ipv6/nc/gnrc_ipv6_nc.c]: Transitive includes: Guaranteed future weird build errors ;).

Comment (cgundogan) [inline sys/net/gnrc/network_layer/ipv6/nc/gnrc_ipv6_nc.c]: addressed

**Comment by cgundogan** [comment]
@authmillenon addressed your comment

**Comment by authmillenon** [comment]
ACK.

**Comment by cgundogan** [comment]
squashed

**Comment by cgundogan** [comment]
travis agrees - GO

**Pull request merged**